dotnet/src/Agents/OpenAI/OpenAIResponseAgentThread.cs

```diff
@@ -6,65 +6,49 @@
 using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.SemanticKernel.ChatCompletion;
 using OpenAI.Responses;
 
 namespace Microsoft.SemanticKernel.Agents.OpenAI;
 
 /// <summary>
-/// Represents a conversation thread for an OpenAI responses-based agent.
+/// Represents a conversation thread for an OpenAI Response API based agent when store is enabled.
 /// </summary>
 [ExcludeFromCodeCoverage]
 public sealed class OpenAIResponseAgentThread : AgentThread
 {
     private readonly OpenAIResponseClient _client;
-    private readonly ChatHistory _chatHistory = new();
     private bool _isDeleted = false;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="OpenAIResponseAgentThread"/> class.
     /// </summary>
     /// <param name="client">The agents client to use for interacting with responses.</param>
-    /// <param name="enableStore">Enable storing messages on the server.</param>
-    public OpenAIResponseAgentThread(OpenAIResponseClient client, bool enableStore = false)
+    public OpenAIResponseAgentThread(OpenAIResponseClient client)
     {
         Verify.NotNull(client);
 
         this._client = client;
-        this.StoreEnabled = enableStore;
     }
 
     /// <summary>
     /// Initializes a new instance of the <see cref="OpenAIResponseAgentThread"/> class that resumes an existing response.
     /// </summary>
     /// <param name="client">The agents client to use for interacting with responses.</param>
     /// <param name="id">The ID of an existing response to resume.</param>
-    /// <param name="enableStore">Enable storing messages on the server.</param>
-    public OpenAIResponseAgentThread(OpenAIResponseClient client, string id, bool enableStore = false)
+    public OpenAIResponseAgentThread(OpenAIResponseClient client, string id)
```

Comment:
Is this experimental? Can we change this? suggest using responseId here, not clear what id this is referring to.

```suggestion
    public OpenAIResponseAgentThread(OpenAIResponseClient client, string responseId)
```

Reply:
Yes, the entire package is experimental, I'll make this change